config/src/test/java/org/springframework/security/config/annotation/method/configuration/EnableReactiveMethodSecurityTests.java

```diff
@@ -589,8 +591,8 @@ public DelegatingReactiveMessageService defaultMessageService() {
 		}
 
 		@Bean
-		public Authz authz() {
```

Comment:
Again please leave existing tests as they are. Instead create only new tests that demonstrate the new functionality.